novawallet/Modules/AssetsSearch/AssetSearch/AssetsSearchPresenter.swift

```diff
@@ -44,9 +44,9 @@ class AssetsSearchPresenter: AssetsSearchPresenterProtocol {
             viewModels.isEmpty,
             !result.state.balanceResults.isEmpty,
             result.state.balanceResults.count >= result.state.allChains.count {
-            view?.didReceiveGroups(state: .empty)
+            view?.didReceiveGroups(state: .init(isFiltered: false, listState: .empty))
```

Comment:
Looks like we need another interface as search currently doesn't have filtering notion